Makefile

```diff
@@ -27,18 +27,38 @@ define color
   @printf '\033[0;34m==== $1 ====\033[0m\n'
 endef
 
-all: bin/ekam-bootstrap deps
+all: bin/ekam
+
+bin/ekam: bin/ekam-bootstrap | deps
 	$(call color,building ekam with ekam)
 	@rm -f bin/ekam
 	@CXX="$(CXX)" CXXFLAGS="-std=c++14 $(CXXFLAGS) -pthread" LIBS="-pthread" bin/ekam-bootstrap -j$(PARALLEL)
+	# Ekam rebuilds all the .o files, included the ones used by `ekam-bootstrap`.
+	# This confuses make, which thinks it needs to rebuild `ekam-bootstrap`.
+	# Tell it that's not necessary; and also tell it that `bin/ekam` is newer than `bin/ekam-bootstrap`.
+	touch bin/ekam-bootstrap
+	touch bin/ekam
 	@test -e bin/ekam && printf "=====================================================\nSUCCESS\nOutput is at bin/ekam\n=====================================================\n"
 
+# NOTE: Needs a full install of Ekam instead of the bootstrap so that LSP is available.
+# To avoid recompiling 3 times, this requires the user to have Ekam already installed.
+continuous:
+	$(call color,building ekam with ekam continuously)
+	@CXX="$(CXX)" CXXFLAGS="-std=c++14 $(CXXFLAGS) -pthread" LIBS="-pthread" ekam -j$(PARALLEL) -n :41315 -c
```

Comment:
Maybe change `ekam` to `$(EKAM)`, and then define `EKAM=ekam` up above, so people can override to a specific path.

Reply:
Sure thing, done.